### DISCUSSION TITLE
Link collector names to an agent table or otherwise index agent names

### DISCUSSION THREAD

**Discussion by themerekat** [Symbiota/community-board#37] [Wish List]
In our Symbiota Support Group meeting on Monday, December 6th, Paula Cushing inquired about being able to index or otherwise link collector names to an agent table. This would enable disambiguation of collectors with, e.g., different spellings or abbreviations.

Upvote this if you'd like to see this happen!

**Comment by mickley** [comment]
@themerekat: 

Of possible usefulness, there was a SPNHC talk by Anton Güntsch a few years ago about using Wikidata for this sort of thing. Apparently many collectors are already entered on Wikidata, and it has an API. 

See:
https://www.idigbio.org/content/spnhc-symposium-rescheduled-june-10-envisioning-collections-management-evolving-biodiversity
